### DISCUSSION TITLE
How to expand sub page section title as default?

### DISCUSSION THREAD

**Discussion by daron4ever** [GitbookIO/community#561] [1. How to] [answered]
Hi,

The subpages are collapsed as default.
How to change to show subpage section titles as default?

**Comment by HVGNc** [comment]
How to create sub pages??

**Reply by daron4ever** [reply]
You can nest pages by dragging and dropping a page below an other in the table of contents. Doing this creates a subpage.
https://docs.gitbook.com/content-editor/editor/content-structure/content-in-a-space

**Comment by taszap** [comment, marked as answer]
Hi @daron4ever - at the moment this isn't possible, but we are tracking it as a feature request internally
